client/src/components/AnnouncementCard.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+export default function AnnouncementCard() {
+  const [isShown, setIsShown] = useState(false);
+  return (
+    <div>
+      <div
+        class="block p-6 shadow-lg bg-white w-full border-b-2"
+        onMouseEnter={() => setIsShown(true)}
+        onMouseLeave={() => setIsShown(false)}
+        style={{
+          backgroundColor: isShown ? "#EAE8E8" : "",
+        }}
+      >
+        <div className="flex">
+          <img
+            src="https://mdbcdn.b-cdn.net/img/new/avatars/2.webp"
+            className="rounded-full w-12"
+            alt="Avatar"
+          />
+          <h5 class="leading-tight font-bold ml-3 mb-2">Admin</h5>
```

Comment:
- [ ] make use of flexbox instead of adding manual margins with different values to center a text
- [ ] wrap the 'Admin' and 'date' in a div and use `flex  items-center  justify-center`
```
<div className="flex justify-center items-center">
  <h5 className="ml-2 font-bold">Admin</h5>
  <p className="ml-2 text-xs">Sept. 21, 2022 at 12:44PM</p>
</div>
```